spec/datadog/tracing/contrib/redis/instrumentation_spec.rb

```diff
@@ -12,8 +12,9 @@
 
   # Redis instance supports 16 databases,
   # the default is 0 but can be changed to any number from 0-15,
-  # to configure support more databases, check `redis.conf`
-  let(:test_database) { (0..15).to_a.sample }
+  # to configure support more databases, check `redis.conf`,
+  # since 0 is the default, the SELECT command would be skipped
+  let(:test_database) { (1..15).to_a.sample }
```

Comment:
Nice job catching this! I'm curious -- why is the non-determinism (e.g. picking a random database) needed for this test? Could we eliminate it entirely?